Sources/Rendering/Core/ImageResliceMapper/index.d.ts

```diff
@@ -165,7 +165,7 @@ export interface vtkImageResliceMapper extends vtkAbstractImageMapper {
 	/**
 	 *
 	 * Enable slab slicing mode and set the slab thickness in world space (mm).
-	 * @param {Boolean} slabThickness The slab thickness in world space (mm). Default: 0. 
+	 * @param {Number} slabThickness The slab thickness in world space (mm). Default: 0.
```

Comment:
Can you also please fix the type issue in this file where setSlabThickness should return a boolean (and same with setSlabTrapezoidIntegration, slabType, slicePlane, and slicePolyData below)? These are all auto-generated setters, so they return true/false based on if the value changed.